Update description of device report view (#2997)
MicrosoftDocs/memdocs: windows-365/enterprise/report-remoting-connection.md

## Device history

> When you select on a particular device in one of the reports, you'll see specific information for that device. The information displayed is for the last 14 days of usage with values aggregated for each day individually. If the user has not logged in to the Cloud PC for a particular day, those days will not show in the graph.
> >[Note]
> >There might days when users login and the graph does display those days. This is due to the way the graph calculations exclude certain edge cases that can provide wrong data.

Reviewer: ```suggestion
When you select a particular device in one of the reports, you'll see specific information for that device. The information displayed is for the last 14 days of usage with values aggregated for each day individually. If the user has not signed in to Cloud PC for a particular day, that day will not show in the graph.

> [!NOTE]
> There might be days when users have signed in and the graph does not display those days. This is due to the way the graph calculations exclude certain Edge cases that can provide incorrect data.
```
